lib/Fhp/Action/GetBalance.php

```diff
@@ -48,21 +48,41 @@ public static function create(SEPAAccount $account, bool $allAccounts = false):
         return $result;
     }
 
+    /**
+     * @deprecated Beginning from PHP7.4 __unserialize is used, then this method is never called
+     */
     public function serialize(): string
     {
-        return serialize([
-            parent::serialize(),
+        return serialize($this->__serialize());
+    }
+
+    public function __serialize(): array
+    {
+        return [
+            parent::__serialize(),
             $this->account, $this->allAccounts,
-        ]);
+        ];
     }
 
+    /**
+     * @deprecated Beginning from PHP7.4 __unserialize is used, then this method is never called
```

Comment:
This isn't true. Even under PHP7.4, data serialized (a) with older PHP versions or (b) with older versions of phpFinTS (which isn't unlikely given our slow release cycles) will still be in "C format" and thus call `Serializable::unserialize()` still.

https://wiki.php.net/rfc/custom_object_serialization#encoding_and_interoperability

This means that we will need to keep this function implementation around for a year or so after this PR here is released.

Reply:
Wow, thank you for this information. I would have missed it. It would be great to mention this in the corresponding PHP docs. I updated the deprecation notices.